sdk/servicebus/Microsoft.Azure.ServiceBus/src/SessionHandlerOptions.cs

```diff
@@ -66,7 +66,7 @@ public TimeSpan MessageWaitTimeout
             }
         }
 
-        /// <summary>Gets or sets the maximum number of existing sessions that the User wants to handle concurrently.</summary>
+        /// <summary>Gets or sets the maximum number of existing sessions that the User wants to handle concurrently. Setting this value to be greater than the max number of active sessions in the service will not increase message throughput.</summary>
```

Comment:
>. Setting  [](start = 116, length = 10)

Lets try to keep the length of the line short. i.e., lets do word wrap if the comment is going too long.